### ISSUE TITLE
OpenStack Provider: Ability to associate floating IP on multiple instances

### ISSUE THREAD

**Issue by kevkha** [hashicorp/terraform#10022]
I'm running Terraform v0.7.9. With the following resource I am able to create multiple instances but only of them gets the FIP associated. Is there a way to associate a new floating IP to the rest of the instances?
```
resource "openstack_compute_instance_v2" "multi_webserver" {
  count = "${var.count}"
  name = "${format("webserver-%02d", count.index+1)}"
  image_name      = "${var.image}"
  flavor_name     = "${var.flavor}"
  key_pair        = "${openstack_compute_keypair_v2.webserver_key.name}"
  security_groups = ["${openstack_compute_secgroup_v2.webserver_secgroup.name}"]
  floating_ip     = "${openstack_compute_floatingip_v2.webserver_pool.address}"
  network {
    uuid = "${openstack_networking_network_v2.webserver_network.id}"
  }
}
```

**Comment by jtopjian** [comment]
@kevkha You'll need to use the `count` argument with the `openstack_compute_floatingip_v2` resource, too. You'll then need to use the `element` function with `count.index` to correlate the instance with the floating IP.

Check the Terraform documentation at https://www.terraform.io/docs/index.html for more info :)

I hope this helps!

**Comment by kevkha** [comment]
Thanks so much @jtopjian. Believe me I checked the doc before filing this issue :)
That wasn't that bad. For those who want to achieve the same. I simply added the `count` to my public resource as suggested above like so
```
count = "${var.count}"
```
I then updated my `floating_ip` line with
```
floating_ip = "${element(openstack_compute_floatingip_v2.webserver_pool.*.address, count.index)}"
```
Last but not least my `output` block should display all floating IPs upon completion.
```
output "address" {
  value = ["${openstack_compute_floatingip_v2.webserver_pool.*.address}"]
}
```
I am a happy camper! :) Thanks again @jtopjian
